src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputEmptyLineSeparatorFormerException.java

```diff
@@ -1,6 +0,0 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
-
-public class InputEmptyLineSeparatorFormerException
-{
-
-}
```

Comment:
Try to rephrase question, I do not understand it.
You can keep names of files as they are now. Just move them to special folder.